aten/src/TH/generic/THTensorMoreMath.cpp

```diff
@@ -275,36 +275,6 @@ static void THTensor_(quicksortdescend)(scalar_t *arr, int64_t *idx, int64_t ele
 #undef MAX_LEVELS
 #undef M_SMALL
 
-void THTensor_(sort)(THTensor *rt_, THLongTensor *ri_, THTensor *t, int dimension, int descendingOrder)
-{
-  dimension = at::maybe_wrap_dim(dimension, t);
-  THArgCheck(dimension >= 0 && dimension < THTensor_(nDimensionLegacyNoScalars)(t), 2, "invalid dimension %d",
```

Comment:
I don't think you have this check in your new implementation, unless I missed it.

Reply:
There is check if values is actually a scalar, see `sort_out` method.

Reply:
But you are right, I'd better insert that check.